connectivity/drivers/mbedtls/TARGET_NUVOTON/CMakeLists.txt

```
@@ -1,4 +1,8 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+add_library(mbed-os-mbedtls-nuvoton INTERFACE)
```

Why not the below?
```suggestion
add_library(mbed-os-mbedtls INTERFACE)
```
If Mbed TLS is included when building for a NUVOTON target the build will include this dir and it should just be part of `mbed-os-mbedtls`, no?